### PR TITLE
add CarePlan resource

### LINKED ISSUE #7: carePlan not exist

### PR DESCRIPTION
**Pull request by ClaudiaAlarconLazo**
add CarePlan resource

### CODE REVIEW

File: src/r4/backbones/ICarePlanActivityDetail.ts
Comment (robertoAraneda) [inline src/r4/backbones/ICarePlanActivityDetail.ts:1]: Falta primitive extensio para kind

File: src/r4/resources/ICarePlan.ts
Comment (robertoAraneda) [inline src/r4/resources/ICarePlan.ts:1]: missing primitive extension for instantiatesCanonical, instantiatesUri

**Review by robertoAraneda — CHANGES_REQUESTED** [review]

File: src/r4/backbones/ICarePlanActivityDetail.ts
Comment (ClaudiaAlarconLazo) [reply src/r4/backbones/ICarePlanActivityDetail.ts:1]: done

File: src/r4/resources/ICarePlan.ts
Comment (ClaudiaAlarconLazo) [reply src/r4/resources/ICarePlan.ts:1]: done

**Review by robertoAraneda — APPROVED** [review]

**Pull request merged**